load-testing/tests/relays_stress_helpers_test.go

```diff
@@ -831,7 +831,7 @@ func (s *relaysSuite) addPendingStakeSupplierMsg(supplier *accountInfo) {
 		supplier.address, // The message signer.
 		supplier.address, // The supplier owner.
```

Comment:
Leaving a comment in a random place.

Feels like we're missing basic (similar) validation to `defeault_rev_share`.

Am I missing something as to why we don't need to check for it explicitly?